RELEASE_NOTES.rst

```diff
@@ -21,6 +21,48 @@
 
 .. towncrier release notes start
 
+Airflow 2.10.4 (2024-12-09)
+---------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+TaskInstance ``priority_weight`` is capped in 32-bit signed integer ranges
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Some database engines are limited to 32-bit integer values. As some users reported errors in
+weight rolled-over to negative values, we decided to cap the value to the 32-bit integer. Even
+if internally in python smaller or larger values to 64 bit are supported, ``priority_weight`` is
+capped and only storing values from -2147483648 to 2147483647. (#43611)
+
+Bug Fixes
+^^^^^^^^^
+
+- Fix stats of dynamic mapped tasks after automatic retries of failed tasks (#44300)
+- Fix wrong display of multi-line messages in the log after filtering (#44457)
+- Allow "/" in metrics validator (#42934) (#44515)
+- Fix gantt flickering (#44488) (#44517)
+- Fix problem with inability to remove fields from Connection form (#40421) (#44442)
+- Check pool_slots on partial task import instead of execution (#39724) (#42693)
+- Avoid grouping task instance stats by try_number for dynamic mapped tasks (#44300) (#44319)
+- Re-queue task when they are stuck in queued (#43520) (#44158)
+- Suppress the warnings where we check for sensitive values (#44148) (#44167)
+- API get_task_instance_try_details returns TaskInstanceHistory schema (#43830) (#44133)
+- Log message source details are grouped (#43681) (#44070)
+- Fix duplication of Task tries in the UI (#43891) (#43950)
+- Add correct mime-type in OpenAPI spec (#43879) (#43901)
+- Disable extra links button if link is null or empty (#43844) (#43851)
+- Prevent using ``trigger_rule="always"`` in a dynamic mapped task (#43810)
```

Comment:
I'm not sure we should do this. Let me mess around a bit here first.

Reply:
Are you saying this PR should not me part of 2.10.4? or just the categorization is wrong?

Reply:
tldr, this is an incomplete fix and the docs change is wrong. (Also, not sure why the newsfrag wasn't in the backport?) I propose, for the sake of time, we remove it from 2.10.4. [More here](https://github.com/apache/airflow/pull/43368#issuecomment-2523903356).

Reply:
Removed it from 2.10.4.